app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -413,9 +413,9 @@ $(document).on('change', '.interface_mac', function(event) {
       .find('.interface_mac')
       .attr('id')
   ) {
-    var interface = $('#interfaceModal').find('.interface_mac');
-    var mac = interface.val();
-    var baseurl = interface.attr('data-url');
+    var interface_ = $('#interfaceModal').find('.interface_mac');
```

Comment:
For my own understanding: why is this needed? Is `interface` now a reserved value or a global variable?

Reply:
It is, and it was a small enough change that I didnt test excatly why the js didnt like us using `interface` . https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#future-proofing_javascript

Reply:
Ah ok. That explains it and IMHO that makes it a candidate to already merge it outside of this PR to make it smaller.

Reply:
https://github.com/theforeman/foreman/pull/9937